pkg/reconciler/inmemorychannel/controller/inmemorychannel_test.go

```diff
@@ -807,6 +922,43 @@ func makeUnknownDeployment() *appsv1.Deployment {
 	return d
 }
 
+func makeEventPolicy(eventPolicyName string) *v1alpha1.EventPolicy {
+	return &v1alpha1.EventPolicy{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "eventing.knative.dev/v1alpha1",
+			Kind:       "EventPolicy",
```

Comment:
Do we need these two? They are usually missing in real objects and if we rely on them, it's probably not working in the real code